etc/fetchers/fetch-sheets.js

```diff
@@ -6,6 +6,34 @@ const fetch = require("cross-fetch");
 function toSnakeCase(text) {
   return text.trim().toLowerCase().replace(" ", "_");
 }
+function toTitleCase(str) {
+  return str
+    .split(" ")
+    .map((word) => word.charAt(0).toUpperCase() + word.slice(1))
+    .join(" ");
+}
+function replaceSpacesWithCamelCase(str) {
+  return str.replace(/\s+/g, (s) => {
+    return s.substring(0, 1).toUpperCase() + s.substring(1);
+  });
+}
+function replaceSpecialCharacterWithSpace(str) {
+  return str.replace(/[^a-zA-Z0-9. ]/g, " ");
+}
+function removeSpaces(str) {
+  return str.replace(/\s+/g, "");
+}
+function convertToKebabCase(str) {
+  return str.replace(/([a-z])([A-Z])/g, "$1-$2").toLowerCase();
+}
+function getSlug(name, index) {
+  const kebabName = convertToKebabCase(
+    removeSpaces(
+      replaceSpecialCharacterWithSpace(replaceSpacesWithCamelCase(name)),
+    ),
+  );
+  return `${kebabName}-${index}`;
+}
```

Comment:
I think it would be better if we can reuse the functions in `lib/string-utils.ts` and add the new functions that we need into that file as well.

What's stopping us from doing it? Is it due to the language difference (JS vs TS)?

Reply:
Yes, I can't simply export `lib/string-utils.ts` because of the language difference. 
Probably work if the configuration is updated or use additional library. But it'll take some time to explore and can affect the other part, so it'll be great if someone can help

Reply:
Got it, I've created a separate issue for it in #144. Thanks!